crates/cxx-qt-build/src/lib.rs

```diff
@@ -256,7 +256,6 @@ impl CxxQtBuilder {
     pub fn new() -> Self {
         let mut qt_modules = HashSet::new();
         qt_modules.insert("Core".to_owned());
-        qt_modules.insert("Gui".to_owned());
```

Comment:
cxx-qt-lib defaults toe enabling qt_gui, but now cxx-qt-build doesn't.

Should we also have a feature on cxx-qt-build that then enables the cxx-qt-lib feature? Or we should default to Core + Gui but have a way to replace/clear the modules ? Maybe `qt_modules(...)` should replace rather than extend ?

Reply:
Lets only enable QtCore for now, then opt in to further modules. This is then similar to CMake and is more explicit.

Reply:
Done